eng/Versions.props

```diff
@@ -9,17 +9,19 @@
     <!-- Opt-in repo features -->
     <UsingToolNetFrameworkReferenceAssemblies>true</UsingToolNetFrameworkReferenceAssemblies>
     <UsingToolSymbolUploader>true</UsingToolSymbolUploader>
-    <MicrosoftBuildVersion>16.7.0</MicrosoftBuildVersion>
-    <MicrosoftBuildTasksCore>16.7.0</MicrosoftBuildTasksCore>
+    <!-- latest versions -->
```

Comment:
Are these versions auto updated by Arcade? If so, please mention it in the comment.

Reply:
They are updated by Darc automatically, yes. Of course, will clarify this in the comment